lourdes_villca/tic_tac_toe/src/main/tictactoe/Board.java

```diff
@@ -0,0 +1,114 @@
+package tictactoe;
```

Comment:
A good practice is
- have copyright
- need to analyze if every method is public.

Reply:
Done